app/components/register.jsx

```diff
@@ -106,7 +106,7 @@ export class Register extends Component {
                   </h2>
                   <div className="form-group">
                     <label htmlFor="dex_title">Title</label>
```

Comment:
Hmmm gotcha. Yeah I can see that could be confusing. Yeah we can just table it for now :+1: